src/Feliz.DaisyUI/Modifiers.fs

```diff
@@ -792,4 +794,24 @@ type color =
     static member inline ringOffsetInfo = prop.className "ring-offset-info"
     static member inline ringOffsetSuccess = prop.className "ring-offset-success"
     static member inline ringOffsetWarning = prop.className "ring-offset-warning"
-    static member inline ringOffsetError = prop.className "ring-offset-error"
\ No newline at end of file
+    static member inline ringOffsetError = prop.className "ring-offset-error"
+
+[<Erase>]
+type shadow =
```

Comment:
we may  want to remove these

Reply:
Are these coming from DaisyUI or Tailwind?

Reply:
Tailwind

Reply:
In that case please remove them. I'd like to have only DaisyUI helpers in this lib 🙏

Reply:
Oops, just pushed but you already merged :D

Reply:
No problem, it's already published! 😄